front/src/services/api-http-interceptor.service.js

```diff
@@ -0,0 +1,44 @@
+import { apiPath } from '@config';
+import axios from 'axios';
+
+const baseURL = apiPath,
+      instance = axios.create({
+        baseURL: baseURL,
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json'
+        }
+      });
+
+instance.interceptors.request.use(
+  config => {
+    const token = JSON.parse(localStorage.getItem('user'));
```

Comment:
why do you take this value directly and not from the store?

Reply:
https://github.com/horba/my-trip/issues/44
point 2

Reply:
got it